parlai/mturk/tasks/acute_eval/example_script.py

```diff
@@ -0,0 +1,54 @@
+from parlai.mturk.tasks.pairwise_dialogue_eval.run import main as run_main, add_args
+
+"""
+Example script for running ACUTE-EVAL.
+The only arguments that need to be modified for this to be run are:
+'dialogs_path':  Path to folder containing logs of all model conversations.
```

Comment:
might be a bit more natural if we made these arguments to `set_args` instead of having to hard code the values in this script

Reply:
I set it up to force users to write a script because we have a lot of really long strings and lists in the arguments, and it would be really ugly to try to parse it from commandline, but open to suggestions

Reply:
I meant you could set as function args in `set_args()` not as CLI args. that way you can just import this function and switch the args instead of having to re-write the body of the function every time.

Reply:
I see. I don't particularly have a problem with that, but it would be no different from what currently exists, just in a slightly different place. In that sense, I feel like it's actually cleaner to have it this way if you do want to set a lot of these args to custom (non-default) values.